hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -82,6 +84,9 @@ const (
 	imageCAPI = "gcr.io/k8s-staging-cluster-api/cluster-api-controller@sha256:9288d085f9fa52bb7186e0de2971945cd192d3a1e849135f7381e4f08fe28442"
 	// This comes from https://console.cloud.google.com/gcr/images/k8s-artifacts-prod
 	imageCAPA = "us.gcr.io/k8s-artifacts-prod/cluster-api-aws/cluster-api-aws-controller:v0.7.0"
+	// This image is built from https://github.com/openshift/cluster-machine-approver/tree/release-4.10
+	imageMachineApprover        = "quay.io/openshift/origin-cluster-machine-approver:4.10.0"
+	imageMachineApproverVersion = "4.10.0"
```

Comment:
why do we need the version to be broken down?

Reply:
see https://github.com/openshift/hypershift/pull/477#discussion_r717615659 . updated to don't use version, and will show version=unknown when user run `oc get co` in guest cluster